holoviews/plotting/bokeh/element.py

```diff
@@ -149,6 +150,8 @@ def __init__(self, element, plot=None, show_labels=['x', 'y'], **params):
         self.current_ranges = None
         super(ElementPlot, self).__init__(element, **params)
         self.handles = {} if plot is None else self.handles['plot']
+        element_ids = self.hmap.traverse(lambda x: id(x), [Element])
```

Comment:
Traversing a holomap should be quick but won't be O(1). Still going to be a lost faster than what we had before though!